flake-template/flake.nix

```diff
@@ -8,42 +8,36 @@
     let
       system = "x86_64-linux";
     in {
-      defaultPackage.${system} = self.packages.${system}.my-microvm;
+      packages.${system}.default = self.nixosConfiguration.config.microvm.declaredRunner;
 
-      packages.${system} = {
-        my-microvm = self.nixosConfigurations.my-microvm.config.microvm.declaredRunner;
-      };
-
-      nixosConfigurations = {
-        my-microvm = nixpkgs.lib.nixosSystem {
-          inherit system;
-          modules = [
-            microvm.nixosModules.microvm
-            {
-              networking.hostName = "my-microvm";
-              users.users.root.password = "";
-              microvm = {
-                volumes = [ {
-                  mountPoint = "/var";
-                  image = "var.img";
-                  size = 256;
-                } ];
-                shares = [ {
-                  # use "virtiofs" for MicroVMs that are started by systemd
-                  proto = "9p";
-                  tag = "ro-store";
-                  # a host's /nix/store will be picked up so that no
-                  # squashfs/erofs will be built for it.
-                  source = "/nix/store";
-                  mountPoint = "/nix/.ro-store";
-                } ];
+      nixosConfiguration = nixpkgs.lib.nixosSystem {
```

Comment:
Is this right? Shouldn't it be `nixosConfigurations.my-microvm = ` ?

Reply:
No, yes. This PR needs changes.

@thkoch2001 Are you going to do that?